src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,101 @@
+import css from './Footer.module.css'
+
+export const Footer = (): JSX.Element => { 
+    return (
+        <footer className={css.footer}>
+            <div className={css.shadowBorder}></div>
+            <div className={css.box}>
+                <div className={css.column}>
+                    <h1 className={css.title}>Help</h1>
+                    <ul className={css.list}>
+                        <li className={css.item}>Payment</li>
+                        <li className={css.item}>Delivery</li>
+                        <li className={css.item}>Returns</li>
+                        <li className={css.item}>Gift Card</li>
+                        <li className={css.item}>Guest purchase</li>
+                        <li className={css.item}>Electronic receipt</li>
+                    </ul>
+                </div>
+                <div className={css.column}>
+                    <h1 className={css.title}>Terms</h1>
+                    <ul className={css.list}>
+                        <li className={css.item}>Privacy policy</li>
+                        <li className={css.item}>Cookie settings</li>
+                        <li className={css.item}>Cookies policy</li>
+                        <li className={css.item}>Site Map</li>
+                        <li className={css.item}></li>
+                        <li className={css.item}></li>
+                    </ul>
+                </div>
+                <div className={css.column}>
+                    <h1 className={css.title}>We are NOVA</h1>
+                    <ul className={css.list}>
+                        <li className={css.item}>About Nova</li>
+                        <li className={css.item}>Join Life</li>
+                        <li className={css.item}>Work with us</li>
+                        <li className={css.item}>Press</li>
+                        <li className={css.item}></li>
+                        <li className={css.item}></li>
+                    </ul>
+                </div>
+                <div className={css.info}>
+                    <div className={css.social}> 
+                        <svg
```

Comment:
please put you svgs into assets folder in the root of the project and import them here